test/test_HighCard.py

```diff
@@ -0,0 +1,19 @@
+from testing_base import *
+
+
+def test_HighCardFunc():
```

Comment:
In my opinion, the test is great. It is concise and clear and yet it is thorough. Great job!

Reply:
Thanks Jon!